Graphics/GraphicsEngineOpenGL/src/GLContextAndroid.cpp

```diff
@@ -342,10 +389,17 @@ void GLContext::Terminate()
 
 void GLContext::UpdateScreenSize()
 {
+    EGLSurface CurrSurface = (surface_ == EGL_NO_SURFACE) ? eglGetCurrentSurface(EGL_DRAW) : surface_;
+
+    if (CurrSurface == EGL_NO_SURFACE)
+    {
+        LOG_ERROR_AND_THROW("Failed to attach to EGLSurface: no active surface");
```

Comment:
We through exception only from constructors during initialization. Since this function can be called at run time, let's change this to error message.

Reply:
Okay. I changed it to log the error and return early.